### DISCUSSION TITLE
Love Ormar!

### DISCUSSION THREAD

**Discussion by bolinocroustibat** [collerek/ormar#194] [General]
Just hopping here quickly to say that I love this project.

I've been trying many ORMs (SQLAlchemy, Peewee and TortoiseORM) with FastAPI in order to find the perfect one but was never really satisfied. This week I have been finally trying Ormar on a quick project and I really liked it. I talked about it to my web Python community and I will definitely choose it for my next FastAPI projects.
I wish it was on the official FastAPI doc as one (if not the) recommended ORM.
Thank you for the great work!

**Comment by collerek** [comment]
Thank you for the really nice words, it really means a lot and help to keep me motivated! 😃 

I wrote it many times already, but always good to repeat, I try to build a community around this project, that way it's always easier to find bugs and attract contributors. So if you can - star the project (I know you already did, thanks!) , and spread the news, thanks!

And that's actually a really good idea with adding section in fastapi docs, don't know if it will be accepted though - there is no gino, tortoise etc. sections, just a pewee one (from whatever reason only this one). 🤔

**Reply by bolinocroustibat** [reply]
There's a Peewee section and a SQLAlchemy section in FastAPI doc. And it would seem FastAPI author @tiangolo doesn't really like Peewee (from what I understands, there are some differences in the way async is implemented), so I don't see why not Ormar couldn't be accepted in the docs! Also it is the ORM that kind of respects the most the FastAPI state of mind

**Reply by avico78** [reply]
@bolinocroustibat - totally agree ,
I think @collerek is doing amazing work - wonder what we can be done in order to to help expose this wonderful package to more people , obviously once more people will use it and recommend it fastapi adopt it and recommended it .
